CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 - remove lib folder before rollup build to avoid stale files
 
+### Fixed
+
+- LTI sort is no more case sensitive
```

Comment:
This is not LTI related, I would say "the Renater IdP sort is case-insensitive" instead